mesontest.py

```diff
@@ -579,7 +580,13 @@ def run(args):
         if options.wrapper:
             print('Must not specify both a wrapper and gdb at the same time.')
             return 1
+        if not shutil.which('gdb'):
+            sys.exit('Requested gdb, but it could not be found.')
 
+    if options.wrapper:
+        wrap_bin = options.wrapper[0] # Not perfect, but the first entry in this must alway exist.
+        if not shutil.which(wrap_bin):
```

Comment:
This will fail on Windows if you specify the complete path without `.exe` at the end (such as with MinGW). We should use `dependencies.ExternalProgram` here.